common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -670,36 +679,48 @@ def test_teams_page_input(self):
         teams = self.create_teams(self.topic, self.TEAMS_PAGE_SIZE + 10, time_between_creation=1)
         self.browse_teams_page.visit()
         self.verify_page_header()
-        self.verify_on_page(1, teams, 'Showing 1-10 out of 20 total', True)
+        self.verify_on_page(self.browse_teams_page, 1, teams, 'Showing 1-10 out of 20 total', True)
         self.browse_teams_page.go_to_page(2)
-        self.verify_on_page(2, teams, 'Showing 11-20 out of 20 total', True)
+        self.verify_on_page(self.browse_teams_page, 2, teams, 'Showing 11-20 out of 20 total', True)
         self.browse_teams_page.go_to_page(1)
-        self.verify_on_page(1, teams, 'Showing 1-10 out of 20 total', True)
+        self.verify_on_page(self.browse_teams_page, 1, teams, 'Showing 1-10 out of 20 total', True)
 
-    def test_navigation_links(self):
+    def test_browse_team_topics(self):
         """
         Scenario: User should be able to navigate to "browse all teams" and "search team description" links.
         Given I am enrolled in a course with a team configuration and a topic
             containing one team
         When I visit the Teams page for that topic
         Then I should see the correct page header
-        And I should see the link to "browse all team"
-        And I should navigate to that link
-        And I see the relevant page loaded
-        And I should see the link to "search teams"
-        And I should navigate to that link
-        And I see the relevant page loaded
+        And I should see the link to "browse teams in other topics"
+        When I should navigate to that link
+        Then I should see the topic browse page
         """
+        self.create_teams(self.topic, self.TEAMS_PAGE_SIZE + 10)
```

Comment:
It doesn't strictly need teams because it clicks on the link at the bottom of the page which is there anyway. I added it when I was trying to get the search link to correctly set the focus, but I then decided it was better to test that in Jasmine.

I'll remove this if it works without the change.